echo/qt/tests/helpers.py

```diff
@@ -0,0 +1,23 @@
+__all__ = [
+    "PYQT5_INSTALLED", "PYQT6_INSTALLED",
+    "PYSIDE2_INSTALLED", "PYSIDE6_INSTALLED",
+    "QTPY_INSTALLED", "QT_INSTALLED",
+    "SKIP_QT_TEST"
+]
+
+
+def package_installed(package):
+    try:
+        __import__(package)
+        return True
+    except ImportError:
+        return False
```

Comment:
It is not necessary to import a package (with its myriad of possible side effects) just to check if it's installed.

```suggestion
def package_installed(package):
    from importlib.util import find_spec
    return find_spec(package) is not None
```

Reply:
Yeah, that is definitely a better way to go. Thanks!